server/src/controllers/christmas-tree/admin.es6

```diff
@@ -28,9 +28,7 @@ const getPermitResult = (permit) => {
   if (permit.christmasTreesForest && permit.christmasTreesForest.timezone) {
     eachPermit.issueDate = moment.tz(permit.updatedAt, permit.christmasTreesForest.timezone).format('MM/DD/YYYY');
 
-    eachPermit.expireDate = moment
-      .tz(permit.permitExpireDate, permit.christmasTreesForest.timezone)
-      .format('MM/DD/YYYY');
+    eachPermit.expireDate = moment.tz(permit.permitExpireDate, permit.christmasTreesForest.timezone).format('MM/DD/YYYY');
```

Comment:
If there are files that ONLY have formatting changes, can we not include them in the PR to reduce the noise a bit?

Reply:
yes, stupid prettier plugin, thought I had admin checked out, reverting / truncating and re-pushing